search-commons/src/test/java/no/unit/nva/search/restclients/responses/ViewingScopeTest.java

```diff
@@ -0,0 +1,16 @@
+package no.unit.nva.search.restclients.responses;
+
+import static no.unit.nva.testutils.RandomDataGenerator.randomUri;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.collection.IsIterableContainingInOrder.contains;
+import org.junit.jupiter.api.Test;
+
+public class ViewingScopeTest {
+
+    @Test
+    void shouldReturnNewViewingScopeWhenSupplyingUris() {
```

Comment:
The `create()` method is used in a different package and therefore a test is necessary here for the test coverage